### PR TITLE
Make error handling more resilient:

### PR DESCRIPTION
**Pull request by MalloZup**
Don't panic on error just log them and skip iteration with continue

### CODE REVIEW

File: ha_cluster_exporter.go [hunk 1]
```diff
@@ -149,7 +151,7 @@ func resetClusterMetrics() {
 	err := prometheus.Register(nodeResources)
 	if err != nil {
 		log.Println("[ERROR]: failed to register NodeResource metric. Perhaps another exporter is already running?")
-		log.Panic(err)
+		log.Println(err)
 	}
```
Comment (stefanotorresi) [inline ha_cluster_exporter.go]: maybe we should return this error instead, and use the logger in the caller (`main()`)?

File: ha_cluster_exporter.go [hunk 2]
```diff
@@ -163,7 +165,7 @@ func resetClusterMetrics() {
 	err = prometheus.Register(clusterNodes)
 	if err != nil {
 		log.Println("[ERROR]: failed to register clusterNode metric. Perhaps another exporter is already running?")
-		log.Panic(err)
+		log.Println(err)
 	}
```
Comment (stefanotorresi) [inline ha_cluster_exporter.go]: maybe we should return this error instead, and use the logger in the caller (`main()`)?

File: ha_cluster_exporter.go [hunk 3]
```diff
@@ -218,7 +221,8 @@ func main() {
 			// todo: reflect if we want to error out. We could just ignore error for resiliance
```
Comment (stefanotorresi) [inline ha_cluster_exporter.go]: I think we can remove this comment now

**Comment by MalloZup** [comment]
Removing comment yep good catch 'I will check if we can return in main error. Since we are continuing the for loop so returning will break it. I will check tomorrow thx for review

**Pull request merged**